test/modules/aws-codedeploy-integration.ts

```diff
@@ -372,6 +373,68 @@ describe('AwsCodedeploy Integration Testing', () => {
   );
 });
 
+describe('Move deployments to another region', () => {
+  it('should fail moving just the deployment group', done =>
+    void query(`
+      UPDATE codedeploy_deployment_group
+      SET region = '${nonDefaultRegion}'
+      WHERE name = '${deploymentGroupName}';
+  `)((e?: any) => {
+      console.log({ e });
```

Comment:
Should these `console.log` lines be removed?

Reply:
Oh, yes I can remove it, I just copied it from another test